src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search-form__lable {
```

Comment:
Be careful when naming classes.
There is a typo in the class name. It should be `search-form__label` instead of `search-form__lable`.